src/algorithm/intersects.rs

```diff
@@ -22,26 +22,25 @@ pub trait Intersects<RHS = Self> {
 }
 
 impl Intersects<LineString> for LineString {
-    // See: https://github.com/brandonxiang/
-    // 		geojson-python-utils/blob/33b4c00c6cf27921fb296052d0c0341bd6ca1af2/geojson_utils.py
+    // See: https://github.com/brandonxiang/geojson-python-utils/blob/33b4c00c6cf27921fb296052d0c0341bd6ca1af2/geojson_utils.py
     fn intersects(&self, linestring: &LineString) -> bool {
         let vect0 = &self.0;
         let vect1 = &linestring.0;
         if vect0.is_empty() || vect1.is_empty() {
             return false;
         }
-        for (p01, p02) in vect0.iter().zip(vect0[1..].iter()) {
-            for (p11, p12) in vect1.iter().zip(vect1[1..].iter()) {
-                let u_b = (p02.lat() - p11.lat()) * (p02.lng() - p01.lng()) -
-                          (p12.lng() - p11.lng()) * (p02.lat() - p01.lat());
+        for (a1, a2) in vect0.iter().zip(vect0[1..].iter()) {
+            for (b1, b2) in vect1.iter().zip(vect1[1..].iter()) {
+                let u_b = (b2.y() - b1.y()) * (a2.x() - a1.x()) -
+                          (b2.x() - b1.x()) * (a2.y() - a1.y());
                 if u_b == 0. {
                     // The point is on boundary
```

Comment:
I didn't ignore this comment. I figure if either of you found out more about what this conditional is doing, one of you can address it in a follow-up PR; not reason to hold this one up.